geopandas/io/tests/test_sql.py

```diff
@@ -467,12 +467,15 @@ def test_write_postgis_without_crs(self, engine_postgis, df_nybb):
         df_nybb.crs = None
         with pytest.warns(UserWarning, match="Could not parse CRS from the GeoDataF"):
             write_postgis(df_nybb, con=engine, name=table, if_exists="replace")
+            write_postgis(df_nybb, con=engine, name=table, if_exists="append")
```

Comment:
can you move this after the assertion `assert target_srid == 0, "SRID should be 0, found %s" % target_srid` and add the code to test the assertion again. This works as is, but it would be nice to keep a test for `if_exists="replace"` only.

Reply:
Perfect! Thanks. Just waiting response for the previous review to proceed commits (now following style guide).